project/src/main/world/Overworld.tscn

```diff
@@ -557,352 +362,253 @@ target_properties = {
+[node name="SpawnButtercup11" parent="World/Obstacles" instance=ExtResource( 17 )]
+position = Vector2( 532.99, 1349.44 )
+id = "buttercup_11"
 
 [node name="SpawnButtercupSeat" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( 2493, 929 )
+position = Vector2( 339.509, 1583.36 )
 id = "buttercup_seat"
 elevation = 80.0
 
-[node name="SpawnButtercupFence" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( 2217.94, 1070.42 )
+[node name="SpawnButtercupFence3" parent="World/Obstacles" instance=ExtResource( 17 )]
+position = Vector2( 1114.3, 1323.42 )
 orientation = 1
 id = "buttercup_fence_3"
 
-[node name="SpawnButtercupFence2" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( 2091.36, 1018.8 )
+[node name="SpawnButtercupFence12" parent="World/Obstacles" instance=ExtResource( 17 )]
+position = Vector2( 987.723, 1271.8 )
 id = "buttercup_fence_12"
 
 [node name="SpawnSePlayer" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( 3251.73, 2165.58 )
+position = Vector2( 1764.82, 1680.06 )
 orientation = 2
 id = "se_player"
 
 [node name="SpawnSeSensei" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( 3403.09, 2222.76 )
+position = Vector2( 1916.18, 1737.24 )
 orientation = 2
 id = "se_sensei"
 
-[node name="EExit" parent="World/Obstacles" instance=ExtResource( 24 )]
-position = Vector2( 8332.17, 874.941 )
-destination_scene_path = "res://src/main/world/Overworld.tscn"
-exit_direction = 1
-player_spawn_id = "w_player"
-sensei_spawn_id = "w_sensei"
-
-[node name="SpawnEPlayer" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( 8198.28, 936.019 )
-orientation = 1
-id = "e_player"
-
-[node name="SpawnESensei" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( 8360.02, 845.01 )
-orientation = 1
-id = "e_sensei"
-
-[node name="WExit" parent="World/Obstacles" instance=ExtResource( 24 )]
-position = Vector2( -3635.54, 1987.22 )
-destination_scene_path = "res://src/main/world/Overworld.tscn"
-exit_direction = 5
-player_spawn_id = "e_player"
-sensei_spawn_id = "e_sensei"
-
-[node name="SpawnWPlayer" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( -3511.97, 1915.56 )
-orientation = 3
-id = "w_player"
-
-[node name="SpawnWSensei" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( -3468.71, 1900.27 )
-orientation = 3
-id = "w_sensei"
-
 [node name="SpawnNwPlayer" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( -848.458, -81.4955 )
+position = Vector2( 932.693, 435.451 )
 id = "nw_player"
 
 [node name="SpawnNwPlayer2" parent="World/Obstacles" instance=ExtResource( 17 )]
-position = Vector2( -954.411, -160.54 )
+position = Vector2( 826.74, 356.406 )
 id = "nw_sensei"
 
 [node name="ChatLetters" parent="World" instance=ExtResource( 3 )]
 
 [node name="ChatIcons" type="Node2D" parent="World"]
+position = Vector2( -107.756, 89.797 )
```

Comment:
Move these chat icons back. Also search for any other references to `-107.756, 89.797` because they're likely new bugs.

Reply:
Moved back to `0, 0`. There are no other references to `-107.756, 89.797`.